integrations/standalone/tests/pageobjects/Theme.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, type Locator } from '@playwright/test';
+import { Switch } from './Switch';
+
+export class Theme {
+  readonly switch: Switch;
+  readonly locator: Locator;
+
+  constructor(parent: Locator) {
+    this.switch = new Switch(parent, { name: 'Theme' });
+    this.locator = parent.getByText('Theme');
+  }
+
+  async toggle() {
+    await this.switch.click();
+  }
+
+  async expectLight() {
+    await expect(this.locator).toHaveCSS('color', 'rgb(27, 27, 27)');
+  }
+
+  async expectDark() {
+    await expect(this.locator).toHaveCSS('color', 'rgb(255, 255, 255)');
+  }

```

Comment:
I would suppose to test that the `html` gets the dark class instead of check some colors.
Because if we change this colors in the ui-components, we need then to fix a test here. But if you only test the class then you assume that the themeprovider does his job and colorize the editor correctly.